src/content/docs/sensitive-info/quick-start/snippets/bun/Step3.js

```diff
@@ -0,0 +1,39 @@
+import arcjet, { sensitiveInfo } from "@arcjet/bun";
+import { env } from "bun";
+
+// This function is called by the `sensitiveInfo` rule to perform custom detection on strings.
```

Comment:
Let's leave the custom detect function out of the quick start and move it to the reference. Quick start is about the bare minimum and then the reference can discuss the advanced options.

Reply:
I've added an example to the bun reference, if you're happy with it I'll copy it across to the other references.

Reply:
Did you push it?

Reply:
@e-moran I assume you did this in https://github.com/arcjet/arcjet-docs/pull/48/commits/d93ee1f8d03825dd7ce4537442d43c1c110c7636 ? The reference example looks good, but the quick start still has the custom detect.

Reply:
That is sorted now.